astropy/convolution/tests/test_convolve_kernels.py

```diff
@@ -25,32 +25,31 @@
 for shape in SHAPES_ODD + NOSHAPE:
     for width in WIDTHS:
 
-        KERNELS.append(Gaussian2DKernel(width,
-                                        x_size=shape[0],
-                                        y_size=shape[1],
-                                        mode='oversample',
-                                        factor=10))
-
-        KERNELS.append(Box2DKernel(width,
-                                   x_size=shape[0],
-                                   y_size=shape[1],
-                                   mode='oversample',
-                                   factor=10))
-
-        KERNELS.append(Tophat2DKernel(width,
-                                      x_size=shape[0],
-                                      y_size=shape[1],
-                                      mode='oversample',
-                                      factor=10))
-        KERNELS.append(Moffat2DKernel(width, 2,
-                                      x_size=shape[0],
-                                      y_size=shape[1],
-                                      mode='oversample',
-                                      factor=10))
+        KERNELS.append(
+            Gaussian2DKernel(
+                width, x_size=shape[0], y_size=shape[1], mode='oversample', factor=10
```

Comment:
Again maybe use the keyword argument for `width` here. This will likely lead to a line break but that's fine...